Library/ViewModels/ProjectPamphletViewModel.swift

```diff
@@ -64,12 +67,21 @@ public final class ProjectPamphletViewModel: ProjectPamphletViewModelType, Proje
           }
       }
 
+    let user = self.viewDidLoadProperty.signal
+      .map { AppEnvironment.current.currentUser }
+      .skipNil()
+
     self.goToRewards = freshProjectAndRefTag
       .takeWhen(self.backThisProjectTappedProperty.signal)
       .map { project, refTag in
         (project, refTag)
       }
 
+    let project = freshProjectAndRefTag
+      .map { project, _ in project }
```

Comment:
We could again be point free: 
```suggestion
      .map(first)
```